supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,29 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Admonitions include:
+
+*NOTE*
```

Comment:
Suggest putting this list of admonitions into a bulleted list. And they probably don't need to be bold.